### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by into1head** [johngodley/redirection#654]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 5.6.33
Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/63.0.3239.84 Chrome/63.0.3239.84 Safari/537.36

Error: Parámetro(s) no válido(s): per_page
Action: redirection/v1/redirect?orderby=id&per_page=2 GET
Code: 400 
Raw: {"code":"rest_invalid_param","message":"Par\u00e1metro(s) no v\u00e1lido(s): per_page","data":{"status":400,"params":{"per_page":"per_page debe estar entre 5 (incluido) y 200 (incluido)"}}}

Error: Parámetro(s) no válido(s): per_page
Action: redirection/v1/group?per_page=2 GET
Code: 400 
Raw: {"code":"rest_invalid_param","message":"Par\u00e1metro(s) no v\u00e1lido(s): per_page","data":{"status":400,"params":{"per_page":"per_page debe estar entre 5 (incluido) y 200 (incluido)"}}}
```

**Comment by johngodley** [comment]
Can you click on 'screen options' at the top right of the page and set the per page limit to 20?

**Comment by johngodley** [comment]
You can also try the 3.1 beta:

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists

**Comment by jmada** [comment]
Thanks!
